hack/dockerized

```diff
@@ -49,8 +49,9 @@ if [ -z "$($KUBEVIRT_CRI volume list | grep ${BUILDER})" ]; then
     $KUBEVIRT_CRI volume create ${BUILDER}
 fi
 
-# Make sure that the output directory exists
+# Make sure that the output directory exists on both sides
 $KUBEVIRT_CRI run -v "${BUILDER}:/root:rw,z" --security-opt "label=disable" --rm ${KUBEVIRT_BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/kubevirt/_out
+[ "$SYNC_OUT" = "true" ] && mkdir -p ${OUT_DIR}
```

Comment:
Let's kust always create the dir. What do you think?

Reply:
Sure, sounds good to me